cp1616/src/cp1616_io_controller.cpp

```diff
@@ -66,6 +83,15 @@ Cp1616IOController::~Cp1616IOController()
   
 }
 
+void operator >> (const YAML::Node &node, ControllerModuleData &module)
+{
+  node["id"] >> module.id;
```

Comment:
What happens if the yaml key `"id"` does not exist.  Does this code handle a badly formatted yaml gracefully?